src/api/v1/types.ts

```diff
@@ -133,6 +134,26 @@ export class AccountObject extends Struct {
     @Struct.field(AccountRexInfo, {optional: true}) rex_info?: AccountRexInfo
 }
 
+@Struct.type('get_block_response')
+export class GetBlockResponse extends Struct {
+    @Struct.field('time_point') timestamp!: TimePoint
+    @Struct.field('name') producer!: Name
+    @Struct.field('uint16') confirmed!: UInt16
+    @Struct.field('checksum256') previous!: Checksum256
+    @Struct.field('checksum256') transaction_mroot!: Checksum256
+    @Struct.field('checksum256') action_mroot!: Checksum256
+    @Struct.field('uint32') schedule_version!: UInt32
+    @Struct.field('any', {optional: true}) new_producers?: any
```

Comment:
`any` is not a type that can be decoded, this will throw if `new_producers` exists in the response

Reply:
I'm going to have to find a block that actually returns a value in here to test with then 🤔

Reply:
https://github.com/greymass/eosio-core/pull/2/commits/093c7eb50dc3572f64d15e27241c734ab0142eee will handle the `new_producers` field appropriately now, and a test is included for block 92565371 on EOS that has a schedule change. 

Need to find examples with the other fields now as well to get rid of the other `any` values I had in there...